src/test/common/commandExecutor.test.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Awesome! This would immensely help with potential regressions with "refactors" :facepunch: